examples/react/react-app/src/App.tsx

```diff
@@ -1,33 +1,47 @@
-import { useComponentValue } from "@dojoengine/react";
-import { Entity } from "@dojoengine/recs";
 import { useEffect, useState } from "react";
 import "./App.css";
-import { Direction } from "./utils";
-import { getEntityIdFromKeys } from "@dojoengine/utils";
 import { useDojo } from "./dojo/useDojo";
+import {
+    Dojo_Starter,
+    Direction,
+    MovesEntity,
+    PositionEntity,
+    isPositionEntity,
+    isMovesEntity,
+} from "./dojo/dojo_starter";
 
 function App() {
-    const {
-        setup: {
-            systemCalls: { spawn, move },
-            clientComponents: { Position, Moves },
-        },
-        account,
-    } = useDojo();
+    const dojo_starter = new Dojo_Starter({
+        rpcUrl: "http://localhost:8545",
+        toriiUrl: "http://localhost:5000",
+    });
```

Comment:
I think we should consider passing in the low level `DojoClient` here rather than instantiating it internally. It will make testing easier, since it will be possible to pass a mocked client, and will allow projects to version the dojo core separately from the typed sdk

Reply:
I'm going to leave it as is, as the current `createDojoConfig` still requests the `manifest.json`, which is something we want to move away from.

Reply:
As is for now that is, will update it later when I'm doing actual functionality work!